package.json

```diff
@@ -464,7 +464,7 @@
         ],
         "walkthroughs": [
             {
-                "id": "azure-get-started",
+                "id": "get-started",
```

Comment:
I'd suggest `azureStart`, `azrgStart` or something like that. `get-started` is too generic and likely to collide with some other extensions' walkthrough.

Reply:
good call - changed to `azrgStart`